client-extensions/liferay-object-collection-filter-extension/src/components/FilterForm.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import NumericField from "./NumericField";
+import BooleanField from "./BooleanField";
+import DateField from "./DateField";
+
+const FilterForm = ({fieldName, fieldType}) => {
+
+    let input = null;
+    if (fieldType == 'numeric') {
```

Comment:
I think it better if we use switch case here 
`let input;

switch (fieldType) {
    case 'numeric':
        input = <NumericField fieldName={fieldName} fieldType={fieldType} />;
        break;
    case 'boolean':
        input = <BooleanField fieldName={fieldName} fieldType={fieldType} />;
        break;
    case 'date':
        input = <DateField fieldName={fieldName} />;
        break;
    default:
        // Handle the case when fieldType doesn't match any of the specified cases
        break;
}
`